tornado-unittests/src/main/java/uk/ac/manchester/tornado/unittests/api/TestIO.java

```diff
@@ -163,14 +163,14 @@ public void testLockObjectsInMemory() {
      * {@link uk.ac.manchester.tornado.api.TornadoExecutionPlan} API to execute on specific device.
      *
      * <p>
-     * The following test reproduces a bug, that when the user sets an array as DataTransferMode.FIRST_EXECUTION,
-     * when using the withDevice API the runtime overrides this setting and copies all arrays on every execution.
+     * The following test reproduces a bug, that when the user sets an array as DataTransferMode.FIRST_EXECUTION
+     * via the withDevice method, the runtime overrides this setting and copies all arrays on every execution.
```

Comment:
Let's open an issue to solve this.

Reply:
This is just a clarification of the comment that described the unit-test.